sdks/go/README.md

```diff
@@ -17,26 +17,87 @@
     under the License.
 -->
 
-# Go SDK
+# Go SDK Overview
 
 The Apache Beam Go SDK is the Beam Model implemented in the [Go Programming Language](https://go.dev/).
 It is based on the following initial [design](https://s.apache.org/beam-go-sdk-design-rfc).
+Below describes requirements, how to run examples, execute tests, and contribute to the Go SDK.
 
-## How to run the examples
+_A note on Beam specific terminology used in this README._
 
-**Prerequisites**: to use Google Cloud sources and sinks (default for
-most examples), follow the setup
-[here](https://beam.apache.org/documentation/runners/dataflow/). You can
-verify that it works by running the corresponding Java example.
+_This README uses minimally necessary Beam related terminology to help you determine requirements, usage and
+contribution to the Go SDK. A [definitions](#definitions) section below provides short definitions for you to achieve
+these aims._
 
-The examples are normal Go programs and are most easily run directly.
-They are parameterized by Go flags.
-For example, to run wordcount on the Go direct runner do:
+# Requirements
 
+Aside from the obvious [go](https:/go.dev), you will need to clone the
+[Beam Repository](https://github.com/apache/beam) on your local machine.
+
+To keep terminal commands clear in this README, the following is assumed:
+
+```
+export BEAM_ROOT=path/to/where/you/clone/beam/repository
+```
+
+Only required to run examples, execute tests, and contribute to the Go SDK, [git clone](https://git-scm.com/docs/git-clone)
+the Beam repository:
+
+```sh
+git clone https://github.com/apache/beam.git $BEAM_ROOT
```

Comment:
Let's not rehash what is already documented on the contribution guide. We shouldn't explain git/github here.

https://beam.apache.org/contribute/get-started-contributing/